src/main/kotlin/no/nav/familie/tilbake/dokumentbestilling/felles/header/TekstformatererHeader.kt

```diff
@@ -2,10 +2,11 @@ package no.nav.familie.tilbake.dokumentbestilling.felles.header
 
 import no.nav.familie.tilbake.dokumentbestilling.felles.Brevmetadata
 import no.nav.familie.tilbake.dokumentbestilling.handlebars.FellesTekstformaterer
+import no.nav.familie.tilbake.organisasjon.OrganisasjonService
 
 object TekstformatererHeader {
 
-    fun lagHeader(brevmetadata: Brevmetadata, overskrift: String): String {
+    fun lagHeader(brevmetadata: Brevmetadata, overskrift: String, organisasjonService: OrganisasjonService): String {
```

Comment:
er det ikke enklere å sende med navnet inn istedenfor servicen?

Reply:
Er litt usiker på den? Får då ein slik kode i TekstformatererHeader
```
                    Institusjon(
                        organisasjonsnummer = it.organisasjonsnummer,
                        navn = organisasjonsnavn!!
                    )
```
Og er ikkje det eigentleg fyfy å bruke !!?

Reply:
Blir ikke helt bra det heller nei, er det noen grunn for at vi ikke legger hele institusjonen i brevMetadata? fungere jo å sende servicen med, men jeg synes det skurrer litt å ha den helt nede i lagHeaderr